### DISCUSSION TITLE
Please add the Option to “Hide” Shortcut’s section In the “Home” Tab.

### DISCUSSION THREAD

**Discussion by Gta211** [community/community#143461] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.183.1 (145298821)
Device: iPhone 14 Pro Max (iOS 18.0.1)
</details>

**Comment by kangarruu** [comment]
@Gta211 Thanks for writing in, I'll file this request with our team for future design exploration!
